hubblestack/extmods/modules/conf_publisher.py

```diff
@@ -0,0 +1,46 @@
+
+import logging
+import hubblestack.splunklogging
+import copy
+import time
+
+log = logging.getLogger(__name__)
+
+def publish():
+    log.info('Started publishing config to splunk')
```

Comment:
Can we use debug log here instead of info? I don't think this is information that we need to log by default.